lib/language.js

```diff
@@ -0,0 +1,857 @@
+'use strict';
+
+var _ = require('lodash');
+
+function init(lang) {
+  
+  function language() {
+    return language;
+  }
+
+  var translations = {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['translations' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/797526280/issues/source?bid=2452379&fileBranchId=2723722#l11)